frontend/src/components/Dialog/style.ts

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
```

Comment:
We probably don't need all of this styling (we could probably use the typography component which inherits styling from `theme.ts` for textual stuff, mui button). 
Let's conduct a joint session regarding this and prettify this together?